packages/components/src/Avatar/Avatar.tsx

```diff
@@ -1,34 +1,106 @@
 import React from 'react';
 import styled from 'styled-components';
 import Imgix from 'react-imgix';
-import { space, color as styledColor, TextColorProps } from 'styled-system';
+import {
+  space,
+  color as styledColor,
+  TextColorProps,
+  border,
+  color,
+} from 'styled-system';
+import { Placeholder, Box, Text } from '..';
+
+const AvatarPlaceholder = styled((props) => <Placeholder {...props} />)`
+  ${({ theme }) =>
+    border({
+      theme,
+      borderWidth: 2,
+      borderColor: 'shades.white',
+      borderStyle: 'solid',
+      borderRadius: '50%',
+    })};
+`;
+
+const DefaultAvatar = styled((props) => <Box {...props} />)`
+  ${({ theme }) =>
+    border({
+      theme,
+      borderWidth: 2,
+      borderColor: 'shades.white',
+      borderStyle: 'solid',
+      borderRadius: '50%',
+    })};
+
+  ${({ theme }) =>
+    color({
+      theme,
+      bg: 'brand.red',
+      color: 'text.inverse',
+    })};
+`;
+
+const StyledText = styled((props) => <Text {...props} />)`
+  user-select: none;
+`;
 
 interface AvatarImageProps {
-  src: string;
+  src?: string;
   size?: number;
   className?: string;
-  alt: string;
+  alt?: string;
+  loading?: boolean;
 }
 
-const AvatarImage = ({ src, size, className, alt }: AvatarImageProps) => {
-  return (
+const AvatarImage = ({
+  src,
+  size,
+  className,
+  alt,
+  loading,
+}: AvatarImageProps) => {
+  const parts = alt?.split(' ').filter(Boolean);
+  const initials =
+    parts && parts.length
+      ? `${parts[0][0]}${parts.length > 1 ? parts[parts.length - 1][0] : ''}`
+      : '?';
+
+  return loading ? (
+    <AvatarPlaceholder height="40px" width="40px" />
+  ) : src ? (
     <Imgix
       src={src}
       className={className}
       imgixParams={{ fit: 'crop', h: size, w: size }}
       width={size}
       height={size}
-      htmlAttributes={{ alt }}
+      htmlAttributes={{ alt: alt || 'Kein Avatar' }}
```

Comment:
Vllt. einfach den Namen des Users bzw. dennoch den alt tag?

Reply:
Habe ich an der Stelle leider nicht explizit, aber alt sollte in der Regel der Name des Users sein 🙂